dask_ml/model_selection/_incremental.py

```diff
@@ -33,35 +33,40 @@ def _partial_fit(model_and_meta, X, y, fit_params):
         A new dictionary with updated information.
     """
     with log_errors():
+        start = time()
         model, meta = model_and_meta
 
         if len(X):
             model = deepcopy(model)
             model.partial_fit(X, y, **(fit_params or {}))
 
         meta = dict(meta)
-        meta['time_step'] += 1
+        meta["partial_fit_calls"] += 1
+        meta["partial_fit_time"] = time() - start
 
         return model, meta
 
 
 def _score(model_and_meta, X, y, scorer):
+    start = time()
     model, meta = model_and_meta
     if scorer:
         score = scorer(model, X, y)
     else:
         score = model.score(X, y)
 
     meta = dict(meta)
-    meta.update(score=score)
+    meta.update(score=score, score_time=time() - start)
     return meta
 
 
 def _create_model(model, ident, **params):
     """ Create a model by cloning and then setting params """
     with log_errors(pdb=True):
         model = clone(model).set_params(**params)
-        return model, {'ident': ident, 'params': params, 'time_step': -1}
+        if "<class 'skorch" in str(type(model)):
+            model.initialize()
```

Comment:
Given the different semantics discussed on the skorch issue tracker how secure do we feel about using it here?

Reply:
I'm okay deleting this given that https://github.com/dnouri/skorch/issues/328 is closed.